src/lib/constants.ts

```diff
@@ -15,10 +15,17 @@ export const DISCORD_PATH = "/discord/" as const
 
 // Config
 export const CONTENT_IMAGES_MAX_WIDTH = 800
-export const LAST_COMMIT_BASE_URL =
-  "https://api.github.com/repos/ethereum/ethereum-org-website/commits"
-export const LAST_DEPLOY_BASE_URL =
-  "https://api.github.com/repos/ethereum/ethereum-org-website/pulls?base=master&state=closed"
+export const GITHUB_BASE_API =
+  "https://api.github.com/repos/ethereum/ethereum-org-website"
+export const GITHUB_COMMITS_URL = GITHUB_BASE_API + "/commits"
+export const GITHUB_LAST_DEPLOY_URL =
+  GITHUB_BASE_API + "/pulls?base=master&state=closed"
+export const GITHUB_AUTH_HEADERS = {
```

Comment:
Don't like too much the idea of adding complex objects in the constants file if possible. I think the constant here is just the token.

I understand that you want to reuse this header but I don't think this is the correct place to do it.

Reply:
Yeah fair enough, maybe this just goes in `utils/gh.ts`

Reply:
For similar reasons to below, I placed this in `utils/fetchGitHubContributors`, and imported it into `utils/gh.ts` for usage. Doing this the other way around caused an error as `fs` is not available in the browser.

Reply:
Posted my thoughts on the Discord channel but I think we should move these types of fetches (client-side) to somewhere else and potentially convert them into hooks.

Reply:
Just refactored into a hook... Brought the auth headers object with it... given the direction we're heading, I think this will work, since there is a decent chance we stop querying GitHub for the deploy date and won't need it there anymore.

Reply:
Nice. Ok, yes, agreed.